attributes/default.rb

```diff
@@ -1,2 +1,3 @@
 default['scrutinizer-whitelist']['source-url'] = 'https://scrutinizer-ci.com/api/meta'
 default['scrutinizer-whitelist']['priority'] = '05'
+default['scrutinizer-whitelist']['ports'] = [22]
```

Comment:
I was thinking that it might be desirable to restrict Scrutinizer's access independently. For example, we use that config to allow access to a variety of ports, but we might only want Scrutinizer to deploy, so granting access to 80 and 443 wouldn't make sense.